src/util.js

```diff
@@ -133,6 +133,25 @@ class Util {
     }
     return obj;
   }
+
+  /**
+   * Trims the text value of an XML text node, i.e. remove leading and trailing whitespaces (space, tab, \r and \n)
+   * @param {string} text value of the XML text node
+   * @returns {string} the trimmed value
+   */
+  static removeXmlWhiteSpaces(text) {
```

Comment:
Couldn't we have used `text.trim()` instead?
It handles both whitespace and line terminators.
[Reference](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trim#return_value)